legate/driver/command.py

```diff
@@ -413,7 +413,10 @@ def cmd_eager_alloc(
 def cmd_ucx(
     config: ConfigProtocol, system: System, launcher: Launcher
 ) -> CommandPart:
-    return ("-ucx:tls_host", "rc,tcp,cuda_copy,cuda_ipc,sm,self")
+    if config.multi_node.ranks > 1 and "ucx" in install_info.networks:
+        return ("-ucx:tls_host", "rc,tcp,cuda_copy,cuda_ipc,sm,self")
```

Comment:
This is not needed anymore.

Reply:
Just clarifying: `cmd_ucx` can then be removed entirely?

Reply:
yes, at this point we can remove `cmd_ucx` everywhere in that file.

Reply:
9d95834957301e54eaa9bfecc25e407e7d3cf6fe